main/main.cpp

```diff
@@ -929,7 +932,8 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 				display_driver = I->next()->get();
 
 				bool found = false;
-				for (int i = 0; i < DisplayServer::get_create_function_count(); i++) {
+				int DS_get_create_function_count = DisplayServer::get_create_function_count();
```

Comment:
Not really any significant benefit, reduces readability, would only matter in extremely performance critical situations